Template_Erorri_Split_2.txt

```diff
@@ -0,0 +1,6 @@
+<2019-4-20> | <16:30> | <17:30> | Coop |  Lorem ipsum A

```

Comment:
Le chiusure <> servono solo a indicare il contenuto della sezione quindi per essere corretta questa riga dovrebbe essere così:
```suggestion
2019-4-20 | 16:30 | 17:30 | Coop |  Lorem ipsum A
```

Reply:
Posso dare una mano per eliminare Le chiusure <> ? 
Editando il file qui direttamente?

Reply:
In via di sviluppo non si cambia sviluppatore del branch, si cambia solo in via eccezionale e questa non lo è